string[] signal

---
string result
Process_Result[] objects

